configure.ac

```diff
@@ -1229,8 +1229,10 @@ AM_CONDITIONAL([EMBEDDED_UNIVALUE],[test x$need_bundled_univalue = xyes])
 AC_SUBST(UNIVALUE_CFLAGS)
 AC_SUBST(UNIVALUE_LIBS)
 
-if test x$enable_bip70 != xno; then
-BITCOIN_QT_PATH_PROGS([PROTOC], [protoc],$protoc_bin_path)
+
+if test x$have_protobuf != xno &&
```

Comment:
I changed this from `test x$have_protobuf = xyes` which fixes builds without pkg-config since `have_protobuf=yes` is not set when protobuf is present on non-pkg-config builds.